src/lib/get-file.js

```diff
@@ -5,31 +5,65 @@ const mime = require('mime');
 
 const getPathByBasePath = require('./paths').getPathByBasePath;
 
+const readFileAsync = (filePath) => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filePath, (err, data) => {
+            if (err) {
+                reject(err);
+            }
+            resolve(data);
+        });
+    });
+};
+
+const existFileAsync = (filePath) => {
```

Comment:
method should returns true|false

Reply:
Promise<true|false>

Reply:
No, this method returns `Promise<string>`. I'll rename it.